patches/0005-Add-OpenSSL-crypto-module.patch

```diff
@@ -31,7 +31,7 @@ Subject: [PATCH] Add OpenSSL crypto module
 
 diff --git a/src/crypto/internal/backend/backend_test.go b/src/crypto/internal/backend/backend_test.go
 new file mode 100644
-index 00000000000..c2c06d3bff8
+index 0000000000..c2c06d3bff
```

Comment:
Why did all the hashes change? It looks like they just dropped the last digit?

Reply:
Interesting... my guess for the count change would be different Git versions or settings between me and @qmuntal. I'll take a look and see if that's likely, and maybe we can make `git-go-patch` set a specific length for the underlying `git` commands.

In general though, the hashes change when the underlying files change upstream, so I tend to ignore the index hash changes altogether. 😄 Later in the PR is an example of one of those:

```diff
- index b7e59d7edb6..994749c10c8 100644
+ index 1482b22d08..5f545148bb 100644
```

In general, if I made an intentional change to a patch file, I'll commit the file's diff and let Git include these index hash changes. I don't `add` patch files that didn't have any intentional changes. Not a great workflow--I want to make the tooling smarter here. Seems a bit tough, though, especially if we want to continue to avoid writing code that actually parses the patch content.